templates/views/search.jade

```diff
@@ -105,6 +105,28 @@ block content
 
                 each submission in searchResults
                     +search-result-component-definition(submission)
+                    
+        div(align='center').search-result-summary
+            | Showing #{firstResultNum} - #{lastResultNum} of #{numResultsTotal} result(s)
+            - var previous = firstResultNum - (limit+1)
+            br
+            if (previous >= 0)
+                a(href=originalUrl + '/?offset='+previous) Previous
+                = ' '
+            - var start = (firstResultNum - (limit*5+1)) < 0               ? 0                     : (firstResultNum - (limit*5+1)) / limit 
```

Comment:
I think some of this could be moved into JavaScript

Reply:
Perhaps.  It is pretty tricky, so I would prefer not to move that right now.  You can log an issue to move it later.